### PR TITLE
monochrome palettes: change wording

### PR DESCRIPTION
**Pull request by AlvaroBurnett**

### CODE REVIEW

File: content/Video_Display.md [hunk 1]
```diff
@@ -179,15 +179,15 @@ instead.
 
 ### FF48 - OBP0 (Object Palette 0 Data) (R/W) - Non CGB Mode Only
 
-This register assigns gray shades for sprite palette 0. It works exactly
-as BGP (FF47), except that the lower two bits aren't used because
-sprite data 00 is transparent.
+This register assigns gray shades to the color indexes of the OBJs that use this palette. It works exactly
+like BGP (FF47), except that the lower two bits aren't used because
```
Comment (ISSOtm) [inline content/Video_Display.md:183]: Maybe "aren't used" could be changed to "are ignored"?

Comment (AlvaroBurnett) [reply content/Video_Display.md]: Yep. Fixed.

Comment (ISSOtm) [reply content/Video_Display.md]: Likewise line 189 for consistency.

Comment (AlvaroBurnett) [reply content/Video_Display.md]: Fixed

**Review by ISSOtm — APPROVED** [review]

**Review by avivace — APPROVED** [review]

**Pull request merged**